packages/design-system-docs/src/pages/components/Tooltip/_Tooltip.docs.scss

```diff
@@ -27,6 +27,8 @@ Style guide: components.tooltip.tooltip
 /*
 `<TooltipIcon>`
 
+When using the `<TooltipIcon>` as the only child of `<Tooltip>`, be sure to provide an `aria-label` to ensure an accessible name for the trigger.
```

Comment:
```suggestion
When using the `<TooltipIcon>` as the only child of `<Tooltip>`, be sure to provide an `aria-label` on the `<Tooltip>` to ensure an accessible name for the trigger.
```

Reply:
I like the idea of updating this documentation. I wonder if it would also be helpful to change the example here to actually be a complete tooltip example, and that would give us a chance to show the `aria-label` in use. Like we could make this like the storybook example. We're also going to be ripping this out soon, so it might not be worth it.